go-controller/pkg/ovn/controller/admin_network_policy/admin_network_policy.go

```diff
@@ -32,6 +49,888 @@ func (c *Controller) processNextANPWorkItem(wg *sync.WaitGroup) bool {
 	return true
 }
 
+// NOTE: Iteration v1 of ANP will only support upto 100 ANPs
+// We will use priority range from 30000 (0) to 20000 (99) ACLs (both inclusive, note that these ACLs will be in tier1)
+// In order to support more in the future, we will need to fix priority range in OVS
+// See https://bugzilla.redhat.com/show_bug.cgi?id=2175752 for more details.
+const (
+	ANPFlowStartPriority            = 30000
+	ANPMaxRulesPerObject            = 100
+	ANPIngressPrefix                = "ANPIngress"
+	ANPEgressPrefix                 = "ANPEgress"
+	ANPExternalIDKey                = "AdminNetworkPolicy" // key set on port-groups to identify which ANP it belongs to
+	OVNKSupportedPriorityUpperBound = 99                   // corresponds to 20100 ACL priority
+)
+
+// TODO: Double check how empty selector means all labels match works
+type adminNetworkPolicySubject struct {
+	namespaceSelector labels.Selector
+	podSelector       labels.Selector
+	// map of namespaces matching the provided namespaceSelector
+	// {K: namespace name; V: {map of pods matching the provided podSelector}}
+	namespaces map[string]map[string]bool
+	// all the LSP port UUIDs of the subject pods selected by this ANP
```

Comment:
> It would be good to explain here the rationale behind caching the UUIDs.
> 
> Because it is apparently not needed as the LSP name is an index as well and we should be able to build it from the pod namespaced name.
> 
> I am guessing this is to add the UUID to the port groups and perhaps we would be saving a lookup in the libovsdb client cache so performance related? I would leave a note here as well whether this is backed up with performance data so or not so it can be considered in the future.

yes correct, though we can use namespace_name as index for LSP, its a pity we don't use LSP names in PG.LSP and use uuids instead...
its currently not backed by perf data will add a TODO. Also the other reason for storing these is.. its easier on every pod/namespace update to get a diff of what previous state was and new state is of the uuid's without having to lookup libovsdb cache each time - so its both perf and coding logic easiness.. will update the comment to be more explicit and call this out.

Reply:
>  Also the other reason for storing these is.. its easier on every pod/namespace update to get a diff of what previous state was and new state is of the uuid's without having to lookup libovsdb cache each time - so its both perf and coding logic easiness.. will update the comment to be more explicit and call this out.

I guess the LSP names would be good enough for this as well